runtime/polkadot/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "polkadot-runtime"
+name = "runtime-polkadot"
```

Comment:
I would keep the `polkadot-runtime`, `kusama-runtime` etc names

Reply:
The reason for renaming them is to provide some straightforward mapping between the location in the directory tree and the package name; improving that mapping is AFAICT one of the major motivations for this issue.